### DISCUSSION TITLE
Add new node(s) to the graph with trained model.

### DISCUSSION THREAD

**Discussion by cappelchi** [danielegrattarola/spektral#400] [Q&A]
Could You suggest, how to add new node(s) to the single graph, that have trained classification model, to make prediction(classification) for this new nodes with existing model?

**Comment by danielegrattarola** [comment]
Sorry for the late reply. 

It should be fairly straightforward, a node is represented as a row of the node features matrix and its edges are represented as a row and column of the adjacency matrix. 
Simply add a row to the node features and a row+column to the adjacency matrix, and feed the new graph to the trained model.
